### ISSUE TITLE
Electronic AOA display still works when mechanical is inop.

### ISSUE THREAD

**Issue by NikolaiVChr** [NikolaiVChr/f16#464]
The AoA indexer should match it

**Comment by ph-jake** [comment]
After reading and trying to interpret GR1F-16CJ-1 "AOA SYSTEM" again, and considering that the static port failure is used to model multiple AoA sensor failure, I want to introduce the following changes to approximate what's described:

- Lock alpha-indicated to 11 deg
- Leave AoA gauge and index operational
- Replace FLCS AOA FAIL by FLCS AOA WARN
- Remove the FPM from HUD

This is what I think the text suggest, apart from the FPM. Not sure what else is/should be affected.

**Comment by Rudolf339** [comment]
I am not that familiar with the details of the f-16, but the aircraft I know use INS velocities for FPM, it's independent of the AoA instrument (or used as backup)

**Comment by ph-jake** [comment]
Yeah, the relation between AoA sensors and FPM felt sketchy to me too. So we can assume that's left in.
